docs/content/intro/quickstart.md

```diff
@@ -199,20 +199,48 @@ brig login --insecure --server https://IP_ADDRESS --root
 
 ## Create a Project
 
-A Brigade [project] defines event handlers, such as the definition of a CI pipeline.
-In this example project, the handler prints a message using a string passed in the event payload.
 
 1. Initialize a new Brigade project with the `brig init` CLI command.
 
     ```
     brig init --id first-project
     ```
+    This will create a project definition like the following in `.brigade/project.yaml`, which defines a handler for the "exec" event, that reads the event payload string and prints it out with "Hello, World!".
```

Comment:
This description isn't accurate. The script that's generated doesn't do anything with the payload.

Reply:
Which means some other commands farther on down might have to be edited as well, as they depict a payload being set, but it's not needed.

Reply:
Oh right. All it does is it defines a handler for the exec event, and echoes hello world right?

Reply:
Yes